types/lru-cache/lru-cache-tests.ts

```diff
@@ -39,7 +39,6 @@ new LRUCache<string, Foo>({
     },
 });
 new LRUCache<string, Foo>({ max: num }); // $ExpectType LRUCache<string, Foo>
-new LRUCache<string, Foo>(); // $ExpectType LRUCache<string, Foo>
```

Comment:
Initializing the cache without options in the constructor is no longer possible since v7